src/main/java/uk/gov/hmcts/dts/fact/model/admin/CourtGeneralInfo.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.hmcts.dts.fact.model.admin;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import uk.gov.hmcts.dts.fact.entity.Court;
+
+@Data
+@AllArgsConstructor
```

Comment:
Do you need this annotation if you have a constructor below that uses all of the arguments?

Reply:
The constructor below is using the court entity object to get all the values. This is used in the service class. The AllArgs annotation is needed for setting up some expected values in tests